Modify some UI elements for space conservation and improved style

acestronautical opened the pull request:
* Convert page layout to select
* Convert source manipulation to select with auto show/hide examples
* Move reset settings and preview buttons to bottom with nicer styling
* Move Files to generate option to File Info section
* Restyle signature info section
* Restyle foldines and cutlines options
* Move Flyleafs section lower

sithel commented:
haven't looked at code yet, just playing around with the page. Will circle back to this later (I'm tired, it's Sunday night) but initial thoughts: 
- I like the repositioning of Generated Files & Reset Settings
- I like the changes to Source Manipulation, but the description UI is not updating correctly when loading values from URLs 
- if we're moving things around, I feel like Fly leaves should be rolled under Source Manipulation and not be it's on section
- I don't like the suppression of the Wacky layouts. It's not expanding when reloading from URLs and... frankly, it hurts my feelings a tiny bit -- my love for that section is what had me working on this project for the last 2 years. I know it's not popular, but it's my baby. I know it's too large/messy and I've tried to make it clear that I'll be cleaning it up next/soon

acestronautical commented:
@sithel I have removed the commit to put the wacky layouts in a details pane. I'm sure that these layouts are very useful to the people who need them as I can't imagine any other tool or process which could produce such a layout. My long term thought was we could do something similar to the source manipulation hiding examples based on select value, and convert the signature formats to a select which reveals more advanced options/inputs/examples based on what you've selected. Forgive me any trespass, I am home sick with covid and just noodling around :P

I believe I have fixed the issue with loading source manipulation examples from URL.

I also think that flyleaves should live elsewhere but not sure I want to tackle that yet. For the moment I figured putting it near the signature info made more sense since that is how I use it, to increase flyleaf count until my signatures are all the same.

sithel commented:
🙏 thanks for changing - you're so speedy! 
Hadn't thought about that use for flyleaf (I mean- I never use it) and it makes total sense- am fine leaving it at the bottom then
> and convert the signature formats to a select

Yessssssss! You read my mind. That is indeed what I was planning/I agree.

Wishing you a speedy recovery & offering condolences on your covid. 
will give it the night for anyone else to chip in ( & because sleepy/still haven't glanced at code) and will try to get to this tomorrow morning

sithel commented on src/utils/clickHandlers.js:
```diff
@@ -7,7 +7,7 @@ import { updateAddOrRemoveCustomPaperOption, updatePaperSelectOptionsUnits } fro
 export function handleGenerateClick(generateEl, book) {
 	generateEl.setAttribute('disabled', true);
-	generateEl.innerText = 'Generating, this may take a little while...';
+	generateEl.innerText = 'Generating PDF...';
```
appreciate the addition of `PDF`, but could you put back the warning of waiting -- it definitely can take a non-trivial moment for larger fic to finish processing and in the modern era of "everything is instant", seems useful to set expectations...

acestronautical replied:
I changed the text back to "Generating, this may take a little while..." but added a fontsize change so the text no longer resizes the button and overflows.

sithel reviewed (approved):
Looks good-- appreciate the initiative to get in there and clean things up
just looking for 2 minor changes-- bumping the version number & putting the "please wait" back in

momijizukamori commented:
Yeah, for context, we've discussed maybe making the wacky layouts a separate section, and then corralling the actual options that go with wacky/standard a little better. We def don't want to vanish them, but Sithel and I approached layouting in very different ways, so getting everything organized nicely has been a little more challenging.
